src/routers/transactions/money/index.ts

```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
 
 const router = createRouter();
 
+/**
+ * @api {get} /transactions/money
+ * @description Returns all money transactions.
+ *              Parameters have no effect on this request.
+ * @apiName GetAllMoneyTransactions
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all money transactions
+ * @apiSuccessExample Success-Response:
+  [
+    {
+      "moneyTransactionUuid": "43c542e4-443e-45fa-91d5-f073161eb216",
+      "datetime": "2018-01-23 04:05:06",
+      "toPersonUuid": "3cad5e7c-5444-4de1-aa81-a7d15acb35f1",
+      "fromPersonUuid": "43c542e4-443e-45fa-91d5-f073161eb216",
+      "amount": 223.42,
```

Comment:
just out of curiosity, in this example, it has a precision and in your other pull request there was a precision of 4. Are we going to be making sure that the precision is consistent? or does it not matter too much?